tests/dummy/app/pods/application/template.hbs

```diff
@@ -15,8 +15,11 @@
       <div class='frost-modal-demo-selector-title'>
         Cookbook
         {{#link-to 'simple'}}Basic{{/link-to}}
+        {{#link-to 'simple-loading'}}Basic - Loading{{/link-to}}
```

Comment:
Ah, I don't think we made a conscious decision about the demo height initially, if you can increase it and satisfy your case of showing the loader I'm fine with that.  I believe it's easier to use the demo for a reference if you don't have to go through multiple demos to get the data you need (less is more).

Not a big deal though, so if you see value in keeping both I won't block it